openpype/hosts/maya/plugins/publish/collect_xgen.py

```diff
@@ -30,12 +30,13 @@ def process(self, instance):
         if data["xgmPalettes"]:
             data["xgmPalette"] = data["xgmPalettes"][0]
 
-        data["xgenConnections"] = {}
+        data["xgenConnections"] = []
         for node in data["xgmSubdPatches"]:
-            data["xgenConnections"][node] = {}
-            for attr in ["transform", "geometry"]:
```

Comment:
Is it correct that we're now **not** traversing the `geometry` attribute?

Reply:
That is correct. It was never used by any of the code.